### DISCUSSION TITLE
Getting a previously unseen error when integrating with thirdweb SDK {code: -32004, message: 'endpoint 62ff3f53852035003a873d84 has passed its daily relay limit'}

### DISCUSSION THREAD

**Discussion by NovaBloq** [WalletConnect/walletconnect-monorepo#3957] [Developer Support] [answered]
I'm getting an error that has popped up for previously functional apps.  does anybody know what this may be referring to?

{code: -32004, message: 'endpoint 62ff3f53852035003a873d84 has passed its daily relay limit'}

happens when trying to call functions that require an approval from the wallet.

**Comment by glitch-txs** [comment, marked as answer]
Hi, it seems to be coming from Blockchain API https://docs.walletconnect.com/cloud/blockchain-api
Try adding more providers into your configuration for fallback purposes.
